src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf"),
+    url("./fonts/Avenir-Heavy.ttf"),
+    url("./fonts/Avenir.ttc");
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.form-one {
+  margin-bottom: 20px;
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+}
+
+.form-two {
+  width: 100%;
+  height: 42px;
+}
+
+.bar {
+  width: 100%;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+}
+
+.bar:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.bar:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.labelform {
+  position: relative;
+}
+
+.placeholderimg {
```

Comment:
do not use camelCase in classNames separate words with `-`

Reply:
agree, use camelCase

Reply:
not fixed